nebula-query-and-search/main/classes/AggregateQuery.cls

```diff
@@ -79,19 +80,48 @@ global class AggregateQuery extends SOQL {
     return this.havingAggregate(aggregateFunction, new SOQL.QueryField(field), operator, value);
   }
 
+  global AggregateQuery havingAggregate(SOQL.Aggregate aggregateFunction, Schema.SObjectField field, SOQL.Operator operator, Object value, String bindWithKey) {
+    return this.havingAggregate(aggregateFunction, new SOQL.QueryField(field), operator, value, bindWithKey);
+  }
+
   global AggregateQuery havingAggregate(SOQL.Aggregate aggregateFunction, SOQL.QueryField queryField, SOQL.Operator operator, Object value) {
-    this.havingConditions.add(aggregateFunction.name() + '(' + queryField + ') ' + SOQL.getOperatorValue(operator) + ' ' + value);
+    return this.havingAggregate(aggregateFunction, queryField, operator, value, null);
+  }
+
+  global AggregateQuery havingAggregate(SOQL.Aggregate aggregateFunction, SOQL.QueryField queryField, SOQL.Operator operator, Object value, String bindWithKey) {
+    this.havingConditions.add(
+      String.format(
+        '{0}({1}) {2} {3}',
+        new List<String> {
+          aggregateFunction.name(),
+          queryField.toString(),
+          SOQL.getOperatorValue(operator),
+          (String.isNotBlank(bindWithKey) ? ':' + bindWithKey : new QueryArgument(value).toString())
```

Comment:
I love how you've kept it backwards compatible with `QueryArgument`, this is great!